upi/metal/config.tf

```diff
@@ -132,13 +132,12 @@ EOF
 
 }
 
-variable "pxe_os_image_url" {
+variable "pxe_kernel_args" {
```

Comment:
can the user pass multiple args? if so can we include an example of that? and make sure it will actually work?

Reply:
I think we should support providing multiple arguments yes; there are various use cases for that, especially debugging.  But I wouldn't block on it.

Reply:
right now they could, I assume by using a space delimited string, I assume there's a better type for this in tfvars?

Reply:
If it is space separated, then can we show that as example in the description.

Reply:
also since you are changing the input variable, can you also update https://github.com/openshift/installer/blob/fa2d5bbe2ba00b36c697de7ff0658f51df36a20a/upi/metal/terraform.tfvars.example

and also updating docs on how to provide the os image url with this new input variable, maybe here https://github.com/openshift/installer/blob/fa2d5bbe2ba00b36c697de7ff0658f51df36a20a/docs/user/metal/install_upi.md#required-kernel-parameters-during-pxe

Reply:
@abhinavdahiya Thanks, 0402482  should have docs updates. If there's a more backwards compatible manner to have made this change let me know if you think I should pursue other options.